core/spree_core.gemspec

```diff
@@ -30,8 +30,8 @@ Gem::Specification.new do |s|
   s.add_dependency 'money', '5.1.1'
   s.add_dependency 'paperclip', '~> 3.4.1'
   s.add_dependency 'paranoia', '~> 2.0'
-  s.add_dependency 'rails', '~> 4.0.0'
-  s.add_dependency 'ransack', '1.0.0'
+  s.add_dependency 'rails', '~> 4.0'
```

Comment:
Is that true? Spree-core works with Rails 4.2?

Reply:
It's looking really good in rails 4.1 and we will bring to OFN and adapt what's not working.
This would make it ready for the rails 4.2 upgrade, otherwise we will need to change it again to open up for rails 4.2.
Do you want to make it more specific for now?
`s.add_dependency 'rails', '~> 4.0', '~> 4.1'`
or
`s.add_dependency 'rails', '~> 4.0', '< 4.2.0'`

Reply:
Okay, so it mostly works and we push the responsibility for the details to OFN. That sounds like a good plan.